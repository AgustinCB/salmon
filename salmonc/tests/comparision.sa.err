Constants: [Integer(1), Integer(2), Integer(3), String(0)]
Instructions: [Instruction { instruction_type: Constant(0), location: 0 }, Instruction { instruction_type: Constant(1), location: 0 }, Instruction { instruction_type: Less, location: 0 }, Instruction { instruction_type: Constant(0), location: 1 }, Instruction { instruction_type: Constant(1), location: 1 }, Instruction { instruction_type: LessEqual, location: 1 }, Instruction { instruction_type: Constant(0), location: 2 }, Instruction { instruction_type: Constant(0), location: 2 }, Instruction { instruction_type: LessEqual, location: 2 }, Instruction { instruction_type: Constant(2), location: 3 }, Instruction { instruction_type: Constant(0), location: 3 }, Instruction { instruction_type: Greater, location: 3 }, Instruction { instruction_type: Constant(2), location: 4 }, Instruction { instruction_type: Constant(0), location: 4 }, Instruction { instruction_type: GreaterEqual, location: 4 }, Instruction { instruction_type: Constant(2), location: 5 }, Instruction { instruction_type: Constant(2), location: 5 }, Instruction { instruction_type: GreaterEqual, location: 5 }, Instruction { instruction_type: Constant(2), location: 6 }, Instruction { instruction_type: Constant(1), location: 6 }, Instruction { instruction_type: NotEqual, location: 6 }, Instruction { instruction_type: Constant(2), location: 7 }, Instruction { instruction_type: Constant(2), location: 7 }, Instruction { instruction_type: NotEqual, location: 7 }, Instruction { instruction_type: Constant(2), location: 8 }, Instruction { instruction_type: Constant(1), location: 8 }, Instruction { instruction_type: Equal, location: 8 }, Instruction { instruction_type: Constant(2), location: 9 }, Instruction { instruction_type: Constant(2), location: 9 }, Instruction { instruction_type: Equal, location: 9 }]
Locations: [Location { address: 0, line: 0 }, Location { address: 0, line: 1 }, Location { address: 0, line: 2 }, Location { address: 0, line: 3 }, Location { address: 0, line: 4 }, Location { address: 0, line: 5 }, Location { address: 0, line: 6 }, Location { address: 0, line: 7 }, Location { address: 0, line: 8 }, Location { address: 0, line: 9 }]
