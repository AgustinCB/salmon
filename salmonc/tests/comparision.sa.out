0 - Bool(true)
1 - Bool(false)
2 - Bool(false)
3 - Bool(true)
4 - Bool(true)
5 - Bool(true)
6 - Bool(true)
7 - Bool(true)
8 - Bool(true)
9 - Bool(true)
